homework/PavelBo/exceptions/knowingExceptions/src/main/java/com/tutrit/java/quickstart/service/Verification.java

```diff
@@ -0,0 +1,70 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.exceptions.ArrayLastValueZeroRuntimeException;
+import com.tutrit.java.quickstart.exceptions.ArrayOddValueException;
+import com.tutrit.java.quickstart.exceptions.ArrayLastValueZeroException;
+import com.tutrit.java.quickstart.exceptions.ArrayOddValueRuntimeException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Verification {
+    Logger log = LoggerFactory.getLogger(Verification.class);
+
+    public void arrayIndexOutOfBound(int i){
+        int[] values = {10, 20, 30};
+        log.info("values[{}] = {}", i, values[i]);
+    }
+
+    public void nullPointer(){
+        int[] values = null;
+        log.info(values.toString());
+    }
```

Comment:
These methods looks like test cases, not business logic. 
And from your testes don't seem to be evident in with circumstances exceptions should be thrown. 
So you'd better move this into tests and from it as test cases;

Please, revise other  methods same way.